coffee-shop-kata-solutions/src/test/java/bnymellon/codekata/coffeeshopkata/CoffeeShopTest.java

```diff
@@ -58,12 +58,20 @@ public void setUp()
     }
```

Comment:
Add Americano to the order, to have a variety of items included - all tests should still pass. Same comment for the other two CoffeeShopTest classes, it should be consistent across all 3 classes.

Reply:
Americano is not a record class so i don't want them to confuse with how to deal with that.

Reply:
It shouldn't have an impact on any of the tests here - the food items for order and receipt methods explicitly say to only cover the food case. A list of order items can have any drinks and any foods.

Reply:
Like you should be able to add Americano to the order in setUp() and all of the tests will pass without being changed.